Travis/pmd/pmd-ruleset.xml

```diff
@@ -11,7 +11,9 @@
 
     <rule ref="rulesets/java/braces.xml"/>
 
-    <rule ref="rulesets/java/codesize.xml"/>
+    <rule ref="rulesets/java/codesize.xml">
+        <exclude name="TooManyMethods" />
```

Comment:
I break this rule only if I describe services. For example, service Scene provides more than ten methods for users.